specification/transport_layer/can.tex

```diff
@@ -23,235 +23,252 @@ \subsection{CAN ID structure}
-    Protocol version    & 1     & 1                 & Protocol version, ignore frame (or use according to a newer specification) if this field is not 1. \\
+
+    Protocol version    & 1     & $1$               & Protocol version, ignore frame
+                                                      (or use according to a newer specification)
+                                                      if this field is not 1. \\
 \end{UAVCANSimpleTable}
 
 \begin{UAVCANSimpleTable}{CAN ID fields for service transfers}{|l l l X|}
     \label{table:can_id_fields_service_transfer}
     Field               & Width & Permitted values  & Description \\
-    Transfer priority   & 3     & [0, 7] (any)      & Section \ref{sec:transfer_prioritization}. \\
-    Service not message & 1     & 1                 & Always one for service transfers. \\
-    Request not response& 1     & \{0, 1\} (any)    & 1 for service request, 0 for service response. \\
-    Service ID          & 9     & [0, 255] (any)    & Service ID of the encoded service data structure
+
+    Transfer priority   & 3     & $[0, 7]$ (any)    & Section \ref{sec:transfer_prioritization}. \\
+
+    Service not message & 1     & $1$               & Always one for service transfers. \\
+
+    Request not response& 1     & $\{0, 1\}$ (any)  & 1 for service request, 0 for service response. \\
+
+    Service ID          & 9     & $[0, 511]$ (any)  & Service ID of the encoded service data structure
                                                       (request or response). \\
-    Destination node ID & 7     & [1, 127]          & Node ID of the destination
+
+    Destination node ID & 7     & $[0, 127]$ (any)  & Node ID of the destination
                                                       (i.e., server for requests, client for responses). \\
-    Source node ID      & 7     & [1, 127]          & Node ID of the origin
+
+    Source node ID      & 7     & $[0, 127]$ (any)  & Node ID of the origin
                                                       (i.e., client for requests, server for responses). \\
-    Protocol version    & 1     & 1                 & Protocol version, ignore frame (or use according to a newer specification) if this field is not 1. \\
+
+    Protocol version    & 1     & $1$               & Protocol version, ignore frame
+                                                      (or use according to a newer specification)
+                                                      if this field is not 1. \\
 \end{UAVCANSimpleTable}
 
 \subsubsection{Transfer priority}
 
 Valid values for priority range from 0 to 7, inclusively,
 where 0 corresponds to the highest priority, and 7 corresponds to the lowest priority.
-Mnemonics for transfer priority levels are provided in the section \ref{sec:transfer_prioritization}.
+Mnemonics for transfer priority levels are provided in the section \ref{sec:transfer_prioritization},
```

Comment:
Isn't this section supposed to be non-normative? If so do we have a way of indicating this yet?

Reply:
Yes, it's supposed to be non-normative. We don't have the right formatting primitives in place yet, we'll fix that later. Right now we should be focused on the content.